package.json

```diff
@@ -30,7 +34,10 @@
     "open-source"
   ],
   "dependencies": {
-    "dotenv": "^16.4.4"
+    "@metamask/providers": "^17.2.0",
```

Comment:
I'm skeptical about this being added in. I don't think we should do anything metamask specific here.

Reply:
Used for typescript typing [here](https://github.com/ubiquity/uusd.ubq.fi/blob/f07a96b90f12d024b704a52b3d447f092c6601af/src/scripts/connect-wallet.ts#L6)